include/seqan3/alphabet/structure/wuss.hpp

```diff
@@ -66,12 +68,14 @@ class wuss : public alphabet_base<wuss<SIZE>, SIZE>
     //!\brief Befriend seqan3::alphabet_base.
     friend base_t;
 
+protected:
+    using typename base_t::rank_type;
+    using typename base_t::char_type;
+
 public:
     using base_t::alphabet_size;
     using base_t::to_rank;
     using base_t::to_char;
-    using typename base_t::rank_type;
-    using typename base_t::char_type;
 
     /*!\name Constructors, destructor and assignment
      * \{
```

Comment:
Why was this again?

Reply:
They aren't public in the other classes.